### PR TITLE
Added math logic

### PR DESCRIPTION
**Pull request by damdafayton**
- Copy the files calculate.js and operate.js into a logic/ directory in your project.
- Analyze the files calculate.js and operate.js, be sure you understand what they do.
- Import the files in your Calculator component.
- Implement the event handlers you need to use the math logic from calculate.js and operate.js in your React component.
- Practice what you have learned until now: use state and props, pass props to child components, use lifecycle methods, lift state up.
- Test your app in the browser. It should be able to perform math operations.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,49 +1,65 @@
...
     super(props);
-    this.state = {};
+    this.state = {
+      calculateObj: {},
+      total: 0,
+    };
+  }
+
+  handleClick(e) {
+    console.log('>> ', this, e.target.firstChild.data);
```
Comment (TedLivist) [inline src/components/Calculator.js]: - [x] It is not good practice to push console logs in the final iteration of your app. It kills performance and can expose confidential data in more sophisticated apps. Kindly remove it.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -1,49 +1,65 @@
...
+    let { calculateObj } = this.state;
+    const button = e.target.firstChild.data;
+    calculateObj = calculate(calculateObj, button);
+    const total = calculateObj.total
+      ? calculateObj.total + (calculateObj.operation || '') + (calculateObj.next || '')
+      : (calculateObj.next || 0);
+    this.setState({ total, calculateObj });
+    console.log(calculateObj);
```
Comment (TedLivist) [inline src/components/Calculator.js]: - [x] Kindly remove console logs.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @damdafayton,

Good job so far!
✔️ Use of lifecycle methods
✔️ Events work as expected
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [x] **In your README, kindly add more detailed instructions on how to set up your app locally**

- [x] **Users need to run `npm install` to install packages before they can run the other commands. Kindly add this.**

- [x] **_Check the comments under the review._**

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Meltrust — APPROVED** [review]
Hi @damdafayton,


## Wow, you did it 🎉
Brilliant
## Thank you for deleting your console.logs, and for fixing your readme, as requested by the previous reviewer 💪 🥇 ㊗️ 
### Well done!




Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**